pubspec.yaml

```diff
@@ -1,15 +1,15 @@
 name: dart_coveralls
-version: 0.4.0
+version: 0.5.0
 author: Axel Christ <adracus@gmail.com>
 description: |-
   Pub package to calculate coverage, format it
   to LCOV and send it to coveralls
 homepage: https://github.com/duse-io/dart-coveralls
 environment:
-  sdk: '>=1.9.0 <2.0.0'
+  sdk: '>=1.9.0 <1.22.0'
```

Comment:
The reason is a [breaking change](https://groups.google.com/a/dartlang.org/forum/#!topic/announce/VxSw-V5tx8k) to the VM service protocol in SDK 1.22.0, that I didn't account for with this PR. I have revisited the case and now got it running on the 1.22 dev branch.

TL;DR: underlying issue fixed, will change back to <2.0.0